build.gradle

```diff
@@ -329,40 +327,70 @@ project('spring-rabbit-test') {
 
 }
 
+configurations {
+	docs
+}
+
+def docResourcesVersion = '0.1.0.RELEASE'
+
+dependencies {
+	docs "io.spring.docresources:spring-doc-resources:${docResourcesVersion}@zip"
+}
+
+task prepareAsciidocBuild(type: Sync) {
+	dependsOn configurations.docs
+	// copy doc resources
+	from {
+		configurations.docs.collect { zipTree(it) }
+	}
+	// and doc sources
+	from "src/reference/asciidoc/"
+	// to a build directory of your choice
+	into "$buildDir/asciidoc/build"
```

Comment:
Isn't it going to sound awkward: `/build/asciidoc/build`?
Can we just live with the `$buildDir/asciidoc` ?

Reply:
That second `build` is to have a working directory for the Asciidoc conversion. I would have called it `working`, but Brian Clozel did that bit. We can change it, of course.

Reply:
I removed the trailing `build`. It works.